steps/workforce/allocations.ts

```diff
@@ -30,9 +30,18 @@ export const allocateCase = async (page: Page, crn: string, allocation: Allocati
     await page.locator('button', { hasText: 'Continue' }).click()
 
     // Confirm allocation
-    await expect(page).toHaveTitle(/.*Allocate to practitioner.*/)
+    await expect(page).toHaveTitle(/.*[A-Z][a-z''-]+([ -][A-Z][A-z''-]+)* | Choose practitioner | Manage a workforce.*/)
```

Comment:
Again it doesn't seem necessary to have this regex here.  You could just omit it.